server/src/test/java/io/crate/integrationtests/SQLHttpIntegrationTest.java

```diff
@@ -49,19 +58,56 @@
 
 public abstract class SQLHttpIntegrationTest extends IntegTestCase {
 
-    private HttpPost httpPost;
+    private final boolean usesSSL;
+    private final SSLContext sslContext;
+
     private InetSocketAddress address;
+    protected URI uri;
+    protected HttpClient httpClient;
 
-    protected final CloseableHttpClient httpClient;
-    private final boolean usesSSL;
+    private static TrustManager[] trustAll = new TrustManager[]{
+        // Workaround to disable hostname verification (+ all other other certificate validaiton)
+        new X509ExtendedTrustManager() {
+
+            public X509Certificate[] getAcceptedIssuers() {
+                return null;
+            }
+
+            public void checkClientTrusted(X509Certificate[] certs, String authType) {
+            }
+
+            public void checkServerTrusted(java.security.cert.X509Certificate[] certs, String authType) {
+            }
+
+            @Override
+            public void checkClientTrusted(X509Certificate[] chain, String authType, Socket socket) throws CertificateException {
+            }
+
+            @Override
+            public void checkServerTrusted(X509Certificate[] chain, String authType, Socket socket) throws CertificateException {
+            }
+
+            @Override
+            public void checkClientTrusted(X509Certificate[] chain, String authType, SSLEngine engine) throws CertificateException {
+            }
+
+            @Override
+            public void checkServerTrusted(X509Certificate[] chain, String authType, SSLEngine engine) throws CertificateException {
+            }
+        }
+    };
 
     public SQLHttpIntegrationTest() {
         this(false);
     }
 
     public SQLHttpIntegrationTest(boolean useSSL) {
-        this.httpClient = HttpClients.custom()
-            .setSSLHostnameVerifier(new NoopHostnameVerifier()).build();
+        try {
+            sslContext = SSLContext.getInstance("TLSv1.2");
```

Comment:
minor: maybe extract the protocol to a static variable, to make it more visible.